packages/expo-router/src/layouts/withLayoutContext.tsx

```diff
@@ -75,6 +75,32 @@ export function useFilterScreenChildren(
 /**
  * Returns a navigator that automatically injects matched routes and renders nothing when there are no children.
  * Return type with `children` prop optional.
+ * 
+ * Enables use of other built-in React Navigation navigators and other navigators built with the React Navigation custom navigator API.
+ *
+ *  @example
+ * ```tsx app/\_layout.tsx
```

Comment:
```suggestion
 * ```tsx app/_layout.tsx
```

I think it might be caused by the escape attempt which break the remark plugin tokenization.

Reply:
![Screenshot 2025-01-23 at 16 40 53](https://github.com/user-attachments/assets/55561808-9a4c-4a61-9dff-ba1c30f59d30)

Data extraction seems to be working fine after the change suggested above.

Reply:
Yep, that's the issue. 

![CleanShot 2025-01-23 at 21 11 13@2x](https://github.com/user-attachments/assets/18d3c0d9-d837-4d97-82e4-a971a0018aca)

Sorry for making the wrong suggestion earlier (markdown text automatically adds "\| in front of "_" 😅 )